test/python/circuit/library/test_evolution_gate.py

```diff
@@ -113,6 +118,27 @@ def test_suzuki_trotter_manual(self):
 
         self.assertEqual(evo_gate.definition.decompose(), expected)
 
+    @data(
+        (X + Y, 0.5, 1, [(Pauli("X"), 0.5), (Pauli("X"), 0.5)]),
+        (X, 0.238, 2, [(Pauli("X"), 1.050420168067227)]),
```

Comment:
This test fails for me locally (the second one only) and I don't quite get why the coefficients of the sampled ops should be different than 1 since we now don't change the coeffs anymore

Reply:
Not sure why it fails for you, maybe it's got something to do with the random seed? The coefficients should be the same but not one (N here is num_gates for us):
![image](https://user-images.githubusercontent.com/5624856/143833287-42f7f527-c926-408a-b432-baba17d3ae24.png)

Reply:
ah maybe making num_gates to an int makes the test fail